apps/web/src/components/layout/components/OrganizationSelect.tsx

```diff
@@ -42,7 +42,7 @@ export default function OrganizationSelect() {
       setLoadingSwitch(true);
       const token = await changeOrganization(organizationId);
       setToken(token);
-      await queryClient.refetchQueries();
+      await queryClient.clear();
```

Comment:
when switching the organization in the dropdown we don't need to refetch the queries but rather clear them all, otherwise some of the queries might be in the fetching state resulting in no loading indicators for example on the workflows page